Automattic/wp-calypso
# Jetpack Pricing | Add pricing breakdown for bundle lightboxes

This PR adds the breakdown of the pricing for Jetpack Bundles - Complete and Security

#### Proposed Changes

* Extract re-usage original price calculation to a hook - `useGetOriginalPrice`
* Create `usePricingBreakdown` to calculate the pricing breakdown
* Create `PricingBreakdown` component to render the UI

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-price-breakdown/ui-logic`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Go to Bundles tab
- Click on "More about Complete" link
- Confirm that you see the pricing breakdown under the checkout CTA
- Confirm that the design matches figma
- Close the modal/lightbox
- Click on "More about Security" link
- Confirm that the above works well for variations
- Confirm the above for Calypso Blue - `/plans/:site`

#### Pre-merge Checklist





Completes 0-as-1203463248826792/f

<img width="1118" alt="Screenshot 2022-11-30 at 7 59 29 PM" src="https://user-images.githubusercontent.com/18226415/204822430-f78c3323-26e2-43ca-97ac-15d79e8ea447.png">
<img width="1106" alt="Screenshot 2022-11-30 at 7 59 39 PM" src="https://user-images.githubusercontent.com/18226415/204822466-850978c4-9151-46a1-a95b-4d6f7619f052.png">
<img width="1117" alt="Screenshot 2022-11-30 at 7 59 46 PM" src="https://user-images.githubusercontent.com/18226415/204822473-c5be1070-5b62-4e88-a978-ac28e8a4bf82.png">
<img width="352" alt="Screenshot 2022-12-01 at 10 50 32 AM" src="https://user-images.githubusercontent.com/18226415/204972111-191c27d3-cbb3-4081-8998-6ec30c1ecfb2.png">


https://user-images.githubusercontent.com/18226415/204822484-239ff835-7115-48b2-81d8-e53d1687db0b.mov



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203463248826792
  - 0-as-1203463248826798
